spec/models/incoming_links_report_spec.rb

```diff
@@ -282,7 +282,7 @@ def stub_empty_referred_topics_data
     it "returns localized titles" do
       stub_empty_referred_topics_data
       expect(top_referred_topics[:title]).to be_present
-      expect(top_referred_topics[:xaxis]).to be_present
```

Comment:
The reason this was "present" is because it was a string: "Missing translation for ...", which misses the intent of the test.